pkg/minishift/addon/manager/addon_manager.go

```diff
@@ -370,6 +370,7 @@ func compareComponentVersions(currentVersion, requiredVersion string, componentN
 			return fmt.Errorf("\nAdd-on does not support %s version %s. "+
 				"You need to use a version %s", componentName, currentVersion, versionRange)
 		}
+		return err
```

Comment:
@praveenkumar This should be `return nil`. We have handled all errors before isn't it.

Reply:
This is consistent with the rest of the file... plus it is a good practice, as you might want to handle the error in the caller.

Reply:
I think returning `nil` when it is not an error is a good programing practice because "explict is better than implicit".  Also I see that we have returned nil in other function in the same file.

Reply:
Discussed with @praveenkumar and it is fine.